packages/angular/src/lib/diagram/diagram.component.ts

```diff
@@ -128,6 +133,21 @@ export class RxZuDiagramComponent
     this.mouseManager ? this.mouseManager.onMouseUp(event) : noop();
   }
 
+  @HostListener('window:keyup', ['$event'])
```

Comment:
this will introduce problems when the library is integrated inside a full application, as it will fetch events from the entire window, instead of only the diagrams scope.

same goes for all other event bindings